.eslintrc.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+    "env": {
+        "commonjs": true,
+        "es6": true
+    },
+    "extends": "eslint:recommended",
+    "rules": {
+        "no-console": 0,
```

Comment:
I mean, I'll change it to error.

Using console gives a different error anyway, `console is not defined`